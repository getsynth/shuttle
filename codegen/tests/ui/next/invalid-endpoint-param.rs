shuttle_codegen::app! {
    #[shuttle_codegen::endpoint(method = get, route = "/goodbye", invalid = bad)]
    async fn goodbye() -> &'static str {
        "Goodbye, World!"
    }
}
