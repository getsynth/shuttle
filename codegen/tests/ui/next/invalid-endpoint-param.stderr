error: invalid endpoint argument

         = help: Only `method` and `route` are valid endpoint arguments.

 --> tests/ui/next/invalid-endpoint-param.rs:2:67
  |
2 |     #[shuttle_codegen::endpoint(method = get, route = "/goodbye", invalid = bad)]
  |                                                                   ^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/next/invalid-endpoint-param.rs:6:2
  |
6 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/next/invalid-endpoint-param.rs`
